OSDOCS-16409: Cluster Admin CQAs (#116629)
openshift/openshift-docs: modules/rosa-nodes-about-autoscaling-nodes-disabling.adoc

// Module included in the following assemblies:
//
// * rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc

:_mod-docs-content-type: CONCEPT
[id="rosa-nodes-about-autoscaling-nodes-disabling_{context}"]
= Disabling autoscaling nodes on a cluster

[role="_abstract"]

> You can disable autoscaling on worker nodes to increase or decrease the number of nodes available by editing the machine pool definition for an existing cluster.

Reviewer: ```suggestion
Disable autoscaling on worker nodes to set a fixed number of nodes available by editing the machine pool definition for an existing cluster.
```
Sounds confusing. We don't disable autoscaling to increase or decrease a number of nodes. But we could probably say:

> Disable autoscaling on worker nodes that increases or decreases the number of nodes available by editing the machine pool definition for an existing cluster.
